src/composables/usePokemons.js

```diff
@@ -1,57 +1,47 @@
-import { ref } from 'vue'
-// import store from '../store'
-import app from '../helpers/axiosConfig'
+import { ref, computed } from 'vue'
+import store from '@/store'
 
 const error = ref(null)
 const loading = ref(false)
-const pokemons = ref(null)
-const pokemon = ref(null)
-
-const pokemonList = ['pikachu', 'ditto', 'charmander']
-function filterByName(value) {
-  for (const x of pokemonList) {
-      if (value.name === x) {
-        console.log(value)
-        return value
-      }
-  }
-}
-
-const getAllPokemons = async () => {
-  error.value = null
-  loading.value = true
-  try {
-    const { data } = await app().get('?limit=132')
-    if(data) {
-      pokemons.value = data.results.filter(filterByName)
-      console.log(pokemons.value)
-    }
-    loading.value = false
-  } catch (err) {
-    error.value = err
-    // const message = await handleResponse(locale, err.response)
-    // store.commit('pushError', message)
-    loading.value = false
-  }
-}
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
 
 const getPokemon = async (name) => {
     error.value = null
     loading.value = true
+    store.commit('pokemons/clearPokemon')
     try {
-      const { data } = await app().get(name)
-      pokemon.value = data.data
+      const pokemons = JSON.parse(localStorage.getItem('pokemons'))
```

Comment:
If you use vuex then rely on it. Here you moved some of the storage logic out from the store.